providers/shared/components/ecs/id.ftl

```diff
@@ -2,6 +2,11 @@
 
 [#assign
     containerChildrenConfiguration = [
+        {
+            "Names" : ["Fragment", "Container"],
```

Comment:
AS this is new, reckon we drop the legacy name and just support "Fragment"

Reply:
Removed